types.go

```diff
@@ -176,9 +176,10 @@ func (f functionInfo) satisfies(method Method) error {
 //
 // If Recorder is nil, no information will be recorded.
 type Settings struct {
-	UseInitialData       bool      // Use supplied information about the conditions at the initial x.
-	InitialFunctionValue float64   // Func(x) at the initial x.
-	InitialGradient      []float64 // Grad(x) at the initial x.
+	UseInitialData       bool            // Use supplied information about the conditions at the initial x.
+	InitialFunctionValue float64         // Func(x) at the initial x.
```

Comment:
Can we remove the "Value" in InitialFunctionValue while we're changing this? The inconsistency is annoying.